src/main/java/com/icthh/xm/ms/dashboard/service/bulk/impl/BulkDashboardServiceImpl.java

```diff
@@ -0,0 +1,101 @@
+package com.icthh.xm.ms.dashboard.service.bulk.impl;
+
+import com.icthh.xm.commons.exceptions.BusinessException;
+import com.icthh.xm.ms.dashboard.domain.Dashboard;
+import com.icthh.xm.ms.dashboard.mapper.DashboardMapper;
+import com.icthh.xm.ms.dashboard.repository.DashboardRepository;
+import com.icthh.xm.ms.dashboard.service.bulk.BulkDashboardService;
+import com.icthh.xm.ms.dashboard.service.dto.BulkDashboardResult;
+import com.icthh.xm.ms.dashboard.service.dto.DashboardDto;
+import java.util.Collection;
+import javax.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class BulkDashboardServiceImpl implements BulkDashboardService {
+
+    private final DashboardMapper dashboardMapper;
+    private final DashboardRepository dashboardRepository;
+
+    @Override
+    public BulkDashboardResult create(Collection<DashboardDto> dashboardItems) {
+
+        BulkDashboardResult saveResult = new BulkDashboardResult();
+
+        dashboardItems
+            .forEach(dashboardEntity -> create(dashboardEntity, saveResult));
+
+        return saveResult;
+    }
+
+    @Override
+    public BulkDashboardResult update(Collection<DashboardDto> dashboardItems) {
+        BulkDashboardResult bulkDashboardResult = new BulkDashboardResult();
+
+        dashboardItems
+            .forEach(dashboardDto -> update(dashboardDto, bulkDashboardResult));
+
+        return bulkDashboardResult;
+    }
+
+    @Override
+    public BulkDashboardResult delete(Collection<DashboardDto> dashboardItems) {
+
+        BulkDashboardResult deleteResult = new BulkDashboardResult();
+
+        dashboardItems
```

Comment:
setup line length like in playbook

Reply:
Already

Reply:
**It's not true**!!! In playbook line length 120!

Reply:
I have length 120 in my cofigs. It is not possible to see here in code my config file. If you are asking about ".forEach" from a new line - it is code formatting for readable code.